go/tasks/plugins/webapi/agent/plugin.go

```diff
@@ -169,11 +171,17 @@ func (p Plugin) Status(ctx context.Context, taskCtx webapi.StatusContext) (phase
 	case admin.State_RETRYABLE_FAILURE:
 		return core.PhaseInfoRetryableFailure(pluginErrors.TaskFailedWithError, "failed to run the job", taskInfo), nil
 	case admin.State_SUCCEEDED:
+		var opReader io.OutputReader
 		if resource.Outputs != nil {
-			err := taskCtx.OutputWriter().Put(ctx, ioutils.NewInMemoryOutputReader(resource.Outputs, nil, nil))
-			if err != nil {
-				return core.PhaseInfoUndefined, err
-			}
+			logger.Infof(ctx, "Agent returned an output")
```

Comment:
Note: remove it once this [pr](https://github.com/flyteorg/flyteplugins/pull/391) is merged.